phpBB/includes/functions_user.php

```diff
@@ -2775,6 +2775,19 @@ function group_user_add($group_id, $user_id_ary = false, $username_ary = false,
 	// Clear permissions cache of relevant users
 	$auth->acl_clear_prefetch($user_id_ary);
 
+	/**
+	* Event after users are added to a group
+	*
+	* @event core.group_add_user_after
+	* @var	int	group_id		ID of the group to which users are added
+	* @var	string group_name		Name of the group
+	* @var	array	user_id_ary		IDs of the users which are added
+	* @var	array	username_ary	names of the users which are added
+	* @since 3.1.7-RC1
+	*/
+	$vars = array('group_id', 'group_name', 'user_id_ary', 'username_ary');
+	extract($phpbb_dispatcher->trigger_event('core.group_add_user_after', compact($vars)));
+
```

Comment:
It would probably be useful to know, whether the users were added as pending (i.e. not really group members yet), as opposed to becoming a full member. So I would add $pending to the variable list.